modules/nodes-cma-autoscaling-custom-prometheus-config.adoc

```diff
@@ -48,7 +49,26 @@ where:
 +
 <service_account>:: Specifies the name of the service account.
 
-. Use the following command to locate the token assigned to the service account:
+. Use the following command to create the service token for the service account:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: thanos-token
+  annotations:
+    kubernetes.io/service-account.name: <service_account>
+  type: kubernetes.io/service-account-token
```

Comment:
@jldohmann this should not be indented.

CC: @ShudiLi, just FYI, I think we missed this in QA.